dotcom-rendering/src/components/TopBarMyAccount.tsx

```diff
@@ -0,0 +1,307 @@
+import { useConfig } from './ConfigContext';
+import type { DropdownLinkType } from './Dropdown';
+import { Dropdown } from './Dropdown';
+
+interface MyAccountProps {
+	mmaUrl: string;
+	idUrl: string;
+	discussionApiUrl: string;
+	idApiUrl: string;
+	authStatus: AuthStatus;
+}
+
+// when SignedIn, authStatus can only be one of the two SignedIn states
+type SignedInProps = MyAccountProps & {
+	authStatus: SignedInWithCookies | SignedInWithOkta;
+	renderingTarget: RenderingTarget;
+};
+
+const myAccountLinkStyles = css`
+	display: flex;
+	align-items: center;
+	${textSansBold17};
+
+	color: ${palette.neutral[100]};
+	transition: color 80ms ease-out;
+	text-decoration: none;
+
+	padding: ${space[1]}px;
+
+	:hover,
+	:focus {
+		text-decoration: underline;
+	}
+
+	svg {
+		fill: currentColor;
+		float: left;
+		height: 18px;
+		width: 18px;
+		margin: 0 ${space[1]}px 0 0;
+	}
+
+	${getZIndex('myAccountDropdown')}
+`;
+
+export const buildIdentityLinks = (
+	mmaUrl: string,
+	idUrl: string,
+	userId: string,
+): DropdownLinkType[] => {
+	/**
+	 * Note: the IDs in here are used by Braze to target notifications so should
+	 * be unique. Please check with Marketing Tools/TX before changing!
+	 */
+	const links = [
+		{
+			id: 'account_overview',
+			url: `${mmaUrl}/`,
+			title: 'Account overview',
+		},
+		{
+			id: 'billing',
+			url: `${mmaUrl}/billing`,
+			title: 'Billing',
+		},
+		{
+			id: 'edit_profile',
+			url: `${mmaUrl}/public-settings`,
+			title: 'Profile',
+		},
+		{
+			id: 'email_prefs',
+			url: `${mmaUrl}/email-prefs`,
+			title: 'Emails & marketing',
+		},
+		{
+			id: 'data_privacy',
+			url: `${mmaUrl}/data-privacy`,
+			title: 'Data privacy',
+		},
+		{
+			id: 'settings',
+			url: `${mmaUrl}/account-settings`,
+			title: 'Settings',
+		},
+		{
+			id: 'help',
+			url: `${mmaUrl}/help`,
+			title: 'Help',
+		},
+		{
+			id: 'comment_activity',
+			url: `${idUrl}/user/id/${userId}`,
+			title: 'Comments & replies',
+		},
+		{
+			id: 'sign_out',
+			url: `${idUrl}/signout`,
+			title: 'Sign out',
+		},
+	] as const;
+
+	return links.map((link) => ({
+		...link,
+		dataLinkName: nestedOphanComponents(
+			'nav3',
+			'topbar',
+			link.id.replaceAll('_', ' '),
+		),
+	}));
+};
+
+const SignIn = ({ idUrl }: { idUrl: string }) => (
+	<a
+		css={myAccountLinkStyles}
+		href={`${idUrl}/signin?INTCMP=DOTCOM_NEWHEADER_SIGNIN&ABCMP=ab-sign-in&${createAuthenticationEventParams(
+			'guardian_signin_header',
+		)}`}
+		data-link-name={nestedOphanComponents('nav3', 'topbar', 'signin')}
+	>
+		<ProfileIcon /> Sign in
+	</a>
+);
+
+export const dropDownOverrides = css`
+	color: ${palette.neutral[100]};
+	padding-right: 0;
+	padding-bottom: 0;
+	margin-top: 0;
+
+	font-weight: bold;
+
+	&:not(ul):hover {
+		color: ${palette.neutral[100]};
+		text-decoration: underline;
+	}
+
+	/** Handles case of *new* top bar being 52px at this breakpoint */
+	${until.tablet} {
+		&:not(button) {
+			top: 48px;
+		}
+	}
+
+	${from.tablet} {
+		right: 0;
+	}
```

Comment:
Could these two declarations be merged? Either designing from `mobile` upwards of `wide` downwards but not both :)